MoneyLoverClone/Scenes/DetailEvent/DetailEventViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  DetailEventViewController.swift
+//  MoneyLoverClone
+//
+//  Created by V000232 on 10/2/20.
+//  Copyright © 2020 Vu Xuan Cuong. All rights reserved.
+//
+
+import UIKit
+
+final class DetailEventViewController: UIViewController {
+    
+    struct Constant {
+        static let numberOfSections = 5
+        static let numberOfRowsInSection = 2
+        static let heightForHeaderInSection: CGFloat = 55
+        static let headerNibName = "HeaderTransactionView"
+    }
+    
+    @IBOutlet weak var tblTransationEvent: UITableView!
```

Comment:
```suggestion
    @IBOutlet weak var tableView: UITableView!
```

Những biến outlet để lên trên đầu